core/blockchain_repair_test.go

```diff
@@ -1832,14 +1832,10 @@ func testRepairWithScheme(t *testing.T, tt *rewindTest, snapshots bool, scheme s
 	}
 	// Force run a freeze cycle
 	type freezer interface {
-		Freeze() error
```

Comment:
Mark: it reverts part of the commit: https://github.com/ethereum/go-ethereum/pull/28683/files#diff-08bfad029bc0c2a3735434672787d567c1f1fb4c21135fc40011fa313e36ac43